### PR TITLE
timer: add urgent

### PR DESCRIPTION
**Pull request by lasers**
timer-urgent

### CODE REVIEW

**Comment by lasers** [comment]
@tobes 🔔 `Timer 0:00:00` to merge this!

File: py3status/modules/timer.py [hunk 1]
```diff
@@ -135,12 +135,19 @@ def make_2_didget(value):
...
             'full_text': self.py3.safe_format(self.format, {'timer': timer})
         }
+        if self.done:
+            response['urgent'] = True
+        return response
 
     def on_click(self, event):
+        if self.done:
+            self.done = False
+            return
```
Comment (tobes) [inline py3status/modules/timer.py]: I don't like the return here it breaks the cancel alarm.

There should be a comment explaining what we are doing.

I think this also breaks the logic of the module.

This module is quite an old one that was a proof of concept for the composites it probably needs some love at some point but actually that is more around threading than anything else so is realated to rewriting some of the py3status core

Comment (lasers) [reply py3status/modules/timer.py]: Okay. We put this on hold. Do you want to take up on this? A diff or something? Not  aware about alarm breakage as I liked `urgent` more.... I guess with `sound` configured, the users will have to click twice to kill the alarm... Maye merging both will alleviate the issue. I will look at this later if you don't want to... as you might want to rewrite `timer.py`. 

And one more thing... at a glance, it looks like `self.alarm` might not be needed here if we used `self.sound` instead. One thing I had issue with composites is that we don't have an good composite example for looping... focused on placeholders, I think... (not that I need it right now, I don't).

**Comment by ultrabug** [comment]
Good to me, thanks mates

**Pull request merged**